packages/widget.vite-plugin/index.html

```diff
@@ -6,9 +6,7 @@
     <title>Palantir: Widget dev mode setup</title>
   </head>
   <body>
-    <div id="root-container">
-      <div id="root"></div>
-    </div>
+    <div id="root"></div>
```

Comment:
This is used for showing the loading spinner before redirect on the dev setup page. The "root-container" part was copied over from the create-app templates which sets up some styles for putting the preview banner on the page but not relevant here so I removed the extra level